core/includes/install.core.inc

```diff
@@ -1454,6 +1450,367 @@ function install_select_language_form($form, &$form_state, $files) {
+    if (!$readable) {
+      $requirements['translations directory readable'] = array(
+        'title'       => st('Translations directory'),
+        'value'       => st('The translations directory is not readable.'),
+        'severity'    => REQUIREMENT_ERROR,
+        'description' => st('The installer requires read permissions to <code>%translations_directory</code> at all times. The <a href="!handbook_url">File permissions and ownership</a> documentation section offers help on this and other topics.', array(
+          '%translations_directory' => $translations_directory,
+          '!handbook_url' => 'https://docs.backdropcms.org/documentation/file-permissions-and-ownership',
+        )),
+      );
+    }
+    // If translations directory is not writable, throw an error.
+    if (!$writable) {
+      $requirements['translations directory writable'] = array(
+        'title'       => st('Translations directory'),
+        'value'       => st('The translations directory is not writable.'),
+        'severity'    => REQUIREMENT_ERROR,
+        'description' => st('The installer requires write permissions to the <code>%translations_directory<code> directory during the installation process. The <a href="!handbook_url">File permissions and ownership</a> documentation section offers help on this and other topics.', array(
+          '%translations_directory' => $translations_directory,
+          '!handbook_url' => 'https://docs.backdropcms.org/documentation/file-permissions-and-ownership',
+        )),
+      );
+    }
+    else {
+      $requirements['translations directory writable'] = array(
+        'title'       => st('Translations directory'),
+        'value'       => st('The translations directory is writable.'),
+      );
+    }
+  }
+
+  // If the translations server can not be contacted, throw an error.
+  if (!$online) {
+    $requirements['online'] = array(
+      'title'       => st('Internet'),
+      'value'       => st('The translation server is offline or cannot be reached.'),
+      'severity'    => REQUIREMENT_ERROR,
+      'description' => st('The installer was unable to contact the translation server to download a translation file. Check your internet connection and verify that your website can reach the translation server at <a href="!server_url">@server_url</a>.', array(
+        '!server_url' => $server_url,
+        '@server_url' => $server_url,
+      )),
+    );
+  }
+  else {
+    $requirements['online'] = array(
+      'title'       => st('Internet'),
+      'value'       => st('The translation server is online.'),
+    );
+    // If translation file is not found at the translation server, throw an
+    // error.
+    if (!$translation_available) {
+      $requirements['translation available'] = array(
+        'title'       => st('Translation'),
+        'value'       => st('The %language translation is not available.', ['%language' => $language]),
```

Comment:
And actually, this also isn't necessary anymore, as we now check for available languages, anyway.

Reply:
Left this in too, because user can select a language to install via URL.

Reply:
Yes, can be triggered by commandline install.